conversions/binary_to_decimal.c

```diff
@@ -1,24 +1,103 @@
+/** 
+ * @file
+ * @brief Converts a binary number to a decimal one.
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂